Scripts/RemediationScripts/Remediate-SecureFTPDeploymentForAppServices.ps1

```diff
@@ -180,8 +189,19 @@ function Enable-SecureFTPDeploymentForAppServices
         [String]
         [Parameter(ParameterSetName = "WetRun", Mandatory = $false, HelpMessage="Specifies the FTP State to be configured")]
         [ValidateSet("FTPSOnly", "Disabled")]
-        $FTPState
+        $FTPState,
+
+        [String]
+        [Parameter(ParameterSetName = "WetRun", HelpMessage="Specifies the path to the file to be used as input for the remediation when AutoRemediation switch is used")]
+        $Path,
+
+        [Switch]
+        [Parameter(ParameterSetName = "WetRun", HelpMessage="Specifies script is run as a subroutine of AutoRemediation Script")]
+        $AutoRemediation,
 
+        [String]
+        [Parameter(ParameterSetName = "WetRun", HelpMessage="Specifies the time of creation of file to be used for logging remediation details when AutoRemediation switch is used")]
+        $TimeStamp
```

Comment:
Line 208: This makes more sense "[Step 1 of 4]Validate and install the modules required to run the script and validating the user"
Take reference from container registry BRS

Reply:
Resolved as per suggestions.